after/config.py

```diff
@@ -3,6 +3,11 @@
 
 @dataclass
 class Paths:
+    """
+    Args:
+        log: Tensorboard log directory
+        data: Input data directory
+    """
```

Comment:
A general idea when using jsonargparse is that people implement their code with proper type hints which are documented in docstrings. These descriptions are automatically used for the help of the CLI. If you run `python main.py --help` you will see these two that I added.

Also worth mentioning that the CLIs include the option `--print_config` which prints the full configuration with all the defaults. This is intended to help people in the creation of their custom configs. I mention this here because if you install the `ruyaml` package and run `python main.py --print_config=comments` the printed config will include the descriptions from the docstrings.